Makefile

```diff
@@ -33,6 +33,8 @@ usage:
 	@echo "Run 'make flash-<board> EXAMPLE=<>' to flash EXAMPLE to a tockloader-supported board."
 	@echo "Run 'make qemu-example EXAMPLE=<>' to run EXAMPLE in QEMU"
 	@echo "Run 'make test' to test any local changes you have made"
+	@echo "Run 'make format' to format any local changes you have made"
```

Comment:
This isn't accurate -- `make format` checks formatting, but does not change formatting.

I don't want `make test` to make edits, so we either need to just update the documentation, or have separate actions for "format the code" and "check formatting".